src/neo/Trie/MPT/MPTDatabase.cs

```diff
@@ -0,0 +1,44 @@
+using Neo.Persistence;
+using System.Text;
+
+namespace Neo.Trie.MPT
+{
+    public class MPTDatabase
+    {
+        private ISnapshot store;
+        public static readonly byte TABLE = 0x4D;
```

Comment:
As we can see, `MPTDatabase` is a key-value storage. If so, can we use `ISnapshot` in `MPTTrie` directly?

Reply:
hmmm. This will move all logic to MTPTrie. Will try

Reply:
public -> private ?